test/NonSiloTests/General/RequestContextTestsNonSiloRequired.cs

```diff
@@ -0,0 +1,393 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.Remoting.Messaging;
+using System.Threading;
+using System.Threading.Tasks;
+using Orleans;
+using Orleans.Runtime;
+using Orleans.Serialization;
+using Xunit;
+
+namespace UnitTests.General
+{
+    public class RequestContextTests_Local : IDisposable
```

Comment:
yes just move things around. since one file contains tests which require a silo and tests which doesn't request a silo